manager.go

```diff
@@ -184,20 +190,20 @@ func (m *Manager) ApplyMigration(ctx context.Context, migration Migration) (err
 
 // filterMigrations applies a filter function that takes the revision of the
 // last applied migration to determine a set of migrations to run.
-func (m *Manager) filterMigrations(ctx context.Context, filter migrationsFilter, verifyHistory bool) ([]Migration, error) {
+func (m *Manager) filterMigrations(ctx context.Context, filter migrationsFilter, verifyHistory bool) (int, []Migration, error) {
 	err := m.EnsureMigrationsTable(ctx)
 	if err != nil {
-		return nil, err
+		return 0, nil, err
 	}
 
 	latest, _, err := m.latestMaybeVerify(ctx, verifyHistory)
 	if err != nil {
-		return nil, err
+		return 0, nil, err
 	}
 
-	migrations, err := filter(latest)
+	pastMigrationCount, migrations, err := filter(latest)
```

Comment:
I'm also OK with a different name for this local (it occurs 18 times). E.g. `skippedMigrations` / `skippedMigrationCount`